### ISSUE TITLE
端侧deploy推理报错：(InvalidArgument) The inverse of Fused batch norm variance should be finite. Found nonfinite values! Please check batch_norm2d_0.w_2

### ISSUE THREAD

**Issue by HustleOoo** [PaddlePaddle/PaddleSeg#3365]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

环境：aarch64+昆仑R200

问题：使用paddleseg训练后得到的pdparams可以正常进行推理，但export导出为pdiparams模型后，使用paddleseg/deploy/python/infer.py进行端侧部署推理时报错如下：
(InvalidArgument) The inverse of Fused batch norm variance should be finite. Found nonfinite values! Please check batch_norm2d_0.w_2 
  [Hint: Expected std::isfinite(variance_array[i]) == true, but received std::isfinite(variance_array[i]):0 != true:1.] (at /workspace/kmis_release/kmis_2/kmis/source/paddle/paddle/fluid/framework/ir/conv_bn_fuse_pass.cc:133)
  说是模型bn层的batch_norm2d_0.w_2值不对，且建议将enable_auto_tune设置为True，我将自动调优功能设置为True之后，还是报这个错误；
  并且我在训练过程中没有出现梯度爆炸、学习率过高（没有出现NaN），我检查了模型bn层的batch_norm2d_0.w_2参数如下：
  
![image](https://github.com/PaddlePaddle/PaddleSeg/assets/119485829/2fb89998-cc02-4324-a637-bd11c0bad1c5)
发现batch_norm2d_0.w_2是个空值，这种情况该如何解决呢？

**Comment by zengshao0622** [comment]
请问具体是哪个模型报错呢？

**Comment by HustleOoo** [comment]
> 请问具体是哪个模型报错呢？

pdiparams推理报错，但是pdparams正常

**Comment by shiyutang** [comment]
请提供给我们动转静导出的指令方便我们测试～

**Comment by HustleOoo** [comment]
> 动转静导出的指令
使用的tools/export.py指令
python tools/export.py \
       --config configs/pp_liteseg/pp_liteseg_stdc1_cityscapes_1024x512_scale0.5_160k.yml \
       --model_path model.pdparams \
       --save_dir output/inference_model

**Comment by HustleOoo** [comment]
> 请提供给我们动转静导出的指令方便我们测试～
使用的官方提供的tools/export.py

python tools/export.py \
       --config configs/pp_liteseg/pp_liteseg_stdc1_cityscapes_1024x512_scale0.5_160k.yml \
       --model_path model.pdparams \
       --save_dir output/inference_model

**Comment by zengshao0622** [comment]
本地测试能够正常导出和推理，建议参考文档：https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.8/docs/model_export_cn.md ，使用官方提供的参数测试一下导出推理是否正常

**Comment by HustleOoo** [comment]
> 本地测试能够正常导出和推理，建议参考文档：https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.8/docs/model_export_cn.md ，使用官方提供的参数测试一下导出推理是否正常

请问下训练指标IoU是否会影响导出模型后的deploy推理？因为之前导出后的pdi模型是可以进行推理的，调整了训练参数后发生了这种报错，精度过低会导致导出模型时bn归一化层的方差计算为NaN？

**Comment by YinRui1991** [comment]
@HustleOoo 请问有解决么，我也遇到了相同的问题。pd模型可以 eval和inference，导出pdi模型后，batch_norm2d的weights里出现了NaN。

**Comment by HustleOoo** [comment]
> @HustleOoo请问有解决么，我也遇到了相同的问题。pd模型可以评估和推理，导出pdi模型后，batch_norm2d的权重里出现了NaN。

没解决，感觉有可能是训练时候的问题，可能是精度太低了，export的时候出了问题

**Comment by TingquanGao** [comment]
Thanks for this issue. As it has been inactive for a long time, we would close it. If you has any questions, please feel free to reopen or new issue, and we will follow up and resolve it.
